cmd2/cmd2.py

```diff
@@ -2470,7 +2472,7 @@ def do_quit(self, _: str) -> bool:
         self._should_quit = True
         return self._STOP_AND_EXIT
 
-    def select(self, opts: Union[str, List[str], List[Tuple[str, Optional[str]]]], prompt: str='Your choice? ') -> str:
+    def select(self, opts: Union[str, List[str], List[Tuple[Any, Optional[str]]]], prompt: str='Your choice? ') -> str:
```

Comment:
This fix to the type hinting here is actually breaking the unit test of the build for the Sphinx documentation.  This needs to be fixed one way or the other.  

Our ReadTheDocs configuration currently builds our Sphinx documentation on Linux using Python 3.5, so that is the platform that should be tested on.  The **invoke** options should have one which builds the Sphinx documentation for testing.